cmd/jailrun/jailrun.go

```diff
@@ -75,32 +74,43 @@ func writeConfig(cfg *jailConfig) error {
 			Src:    proto.String("/srv"),
 			Dst:    proto.String("/"),
 			IsBind: proto.Bool(true),
-			Nosuid: proto.Bool(true),
 			Nodev:  proto.Bool(true),
+			Nosuid: proto.Bool(true),
 		}},
 		Hostname: proto.String("app"),
 		Cwd:      proto.String("/app"),
 		ExecBin: &nsjail.Exe{
 			Path: proto.String("/app/run"),
 		},
 	}
-	if cfg.cgroup.cgroup2 {
+	if cgroup.cgroup2 {
 		msg.UseCgroupv2 = proto.Bool(true)
 		msg.Cgroupv2Mount = proto.String(cgroupPath + "/unified/run")
 	} else {
 		msg.CgroupPidsMount = proto.String(cgroupPath + "/pids")
-		msg.CgroupPidsParent = &cfg.cgroup.pids.parent
+		msg.CgroupPidsParent = &cgroup.pids.parent
 		msg.CgroupMemMount = proto.String(cgroupPath + "/mem")
-		msg.CgroupMemParent = &cfg.cgroup.mem.parent
+		msg.CgroupMemParent = &cgroup.mem.parent
 		msg.CgroupCpuMount = proto.String(cgroupPath + "/cpu")
-		msg.CgroupCpuParent = &cfg.cgroup.cpu.parent
+		msg.CgroupCpuParent = &cgroup.cpu.parent
 	}
-	if cfg.Pow != 0 {
+	if cfg.Pow > 0 {
 		msg.Bindhost = proto.String("127.0.0.1")
 		msg.Port = proto.Uint32(cfg.Port + 1)
 		msg.MaxConns = proto.Uint32(0)
 		msg.MaxConnsPerIp = proto.Uint32(0)
 	}
+	if cfg.TmpSize > 0 {
+		msg.Mount = append(msg.Mount, &nsjail.MountPt{
+			Dst:     proto.String("/tmp"),
+			Fstype:  proto.String("tmpfs"),
+			Rw:      proto.Bool(true),
+			Options: proto.String(fmt.Sprintf("size=%d", cfg.TmpSize)),
+			Nodev:   proto.Bool(true),
+			Nosuid:  proto.Bool(true),
+			Noexec:  proto.Bool(true),
```

Comment:
I feel like these defaults would be unexpected to many users, since it's not standard to disable suid+exec and also not documented or configurable. IMO we should either default to allowing suid and exec without configuration options or add configuration knobs for these settings.

As-is, a workaround would be setting `JAIL_TMP_SIZE` to 0 and then adding the tmpfs mount yourself via hook.sh, but that seems very suboptimal.